src/load_med.cpp

```diff
@@ -894,7 +894,7 @@ BOOL CSoundFile::ReadMed(const BYTE *lpStream, DWORD dwMemLength)
 					DWORD cmdexttable = bswapBE32(pbi->cmdexttable);
```

Comment:
All structs that are reading directly off of `lpStream` can potentially be unaligned, as `lpStream` itself might be unaligned. This struct is `&(lpStream + dwBlockInfo)`, and `dwBlockInfo` *also* isn't guaranteed to be 32-bit aligned.

I think unfortunately there are going to be a lot more of these :(

Reply:
> All structs that are reading directly off of `lpStream` can potentially be unaligned, as `lpStream` itself might point to unaligned memory.

Indeed.

> This struct is `&(lpStream + dwBlockInfo)`, and `dwBlockInfo` _also_ isn't guaranteed to be 32-bit aligned.
> 
> I think unfortunately there are going to be a lot more of these :(

Missed those ones in load_med.c completely. And yes, the whole situation is messy and hard to manage.

Reply:
Working on load_med.cpp - will push an additional patch for it later.